### PR TITLE
Add unicode support

### PR DESCRIPTION
**Pull request by BG4444**
Add unicode support

### CODE REVIEW

**Review by lvu — COMMENTED** [review]
Hi!

What's the point of "--body"? And what is 0x3f?

**Comment by BG4444** [comment]
--body - to generate only body of the html page - it is usable for display rtf as part of web page.

-0x3f is '?' sign.

**Comment by lvu** [comment]
0x3f – then why do return an empty string for it?

**Comment by BG4444** [comment]
M$ Word generate unicode string as \?\uxxxx\?\uxxxx...

So, we need to remove \? and transcode uxxx to unicode


Valentin Lavrinenko <notifications@github.com> 9 марта 2018 г. 5:05:35 ПП 
написал:

**Comment by BG4444** [comment]
Thx!

File: rtf2html.cpp [hunk 1]
```diff
@@ -246,7 +254,7 @@ int main(int argc, char **argv)
                // font table
                case rtf_keyword::rkw_fonttbl: 
                {
-                  font fnt;
+                    font fnt;
```
Comment (Jojo-Schmitz) [inline rtf2html.cpp:257]: whitespace damage ;-)

**Pull request merged**